cargo-shuttle/src/main.rs

```diff
@@ -50,12 +52,13 @@ impl Shuttle {
         self.ctx.set_api_url(args.api_url);
 
         match args.cmd {
-            Command::Deploy(deploy_args) => self.deploy(deploy_args).await,
-            Command::Status => self.status().await,
-            Command::Logs => self.logs().await,
-            Command::Delete => self.delete().await,
             Command::Auth(auth_args) => self.auth(auth_args).await,
+            Command::Delete => self.delete().await,
+            Command::Deploy(deploy_args) => self.deploy(deploy_args).await,
+            Command::Init(init_args) => self.init(init_args).await,
             Command::Login(login_args) => self.login(login_args).await,
+            Command::Logs => self.logs().await,
+            Command::Status => self.status().await,
```

Comment:
The ordering is nice. Is there a reason you decided to order them other than for readability?
I'm thinking, since Rust will match them in order, maybe we would rather want the most popular commands to be first?

Reply:
That makes sense, too. I was more thinking about alphabetical order in case one wants to find a command when one doesn't know a lot about the commands. However, I think you're probably right that, if the coder is familiar with everything, order by popularity is better :)